xrpl/asyncio/transaction/main.py

```diff
@@ -295,6 +301,8 @@ async def _calculate_fee_per_transaction_type(
     Args:
         transaction: the Transaction to be submitted.
         client: the network client with which to submit the transaction.
+        signers_count: the expected number of signers for this transaction.
+            Only used for multisigned transactions.
```

Comment:
The `signers_count` needs to be added to all the methods that call `autofill` as well, like `safe_sign_and_autofill`

Reply:
xrpl.js doesn't seem to include `signers_count` on all methods that call upon `autofill`. Also, I'm not sure if would ever call `safe_sign_and_autofill` if we are doing multisigning

Reply:
Good point